.vuepress/config.js

```diff
@@ -30,7 +30,7 @@ module.exports = {
     editLinkText: '',
     lastUpdated: false,
     smoothScroll: true,
-    nextLinks: false,
+    nextLinks: true,
```

Comment:
why was this enabled?

Reply:
I enabled this to make the flow between the manual pages easier. I can remove if it isn't a change you like.